contracts/farm/FeeExtractAndDistribute_ETH.sol

```diff
@@ -3,51 +3,65 @@
  * Licensed under the Apache License, Version 2.0.
  */
 
-pragma solidity 0.6.12;
+pragma solidity ^0.8.0;
 pragma experimental ABIEncoderV2;
 
-import "@openzeppelin-3.4.0/token/ERC20/SafeERC20.sol";
-import "../interfaces/IERC20Burnable.sol";
-import "./interfaces/Upgradeable.sol";
-import "./interfaces/IWethERC20.sol";
 import "./interfaces/IUniswapV2Router.sol";
 import "../../interfaces/IBZx.sol";
-import "./interfaces/IMasterChefPartial.sol";
-import "./interfaces/IPriceFeeds.sol";
-import "../../interfaces/IStaking.sol";
+import "../../interfaces/IPriceFeeds.sol";
+import "../../interfaces/IStakingV2.sol";
 import "./../staking/interfaces/ICurve3Pool.sol";
+import "@openzeppelin-4.3.2/token/ERC20/IERC20.sol";
+import "../governance/PausableGuardian_0_8.sol";
+
+interface IBridge {
+    function send(
+        address _receiver,
+        address _token,
+        uint256 _amount,
+        uint64 _dstChainId,
+        uint64 _nonce,
+        uint32 _maxSlippage
+    ) external;
+}
 
-contract FeeExtractAndDistribute_ETH is Upgradeable {
-    using SafeMath for uint256;
-    using SafeERC20 for IERC20;
+interface IERC {
+    function approve(address spender, uint amount) external;
```

Comment:
isn't this in here already import "@openzeppelin-4.3.2/token/ERC20/IERC20.sol";

Reply:
approve on USDT does not have a return but IERC20 specifies one so it always fails.

Reply:
> approve on USDT does not have a return but IERC20 specifies one so it always fails.

its a good practice to use safeApprove whenever possible this handles it